packages/server/lib/controllers/provider.controller.ts

```diff
@@ -0,0 +1,46 @@
+import type { NextFunction, Request, Response } from 'express';
+import { configService } from '@nangohq/shared';
+
+class ProviderController {
+    /**
+     * Webapp
+     */
+
+    async listProviders(req: Request, res: Response, next: NextFunction) {
+        try {
+            let templates = configService.templates ?? {};
+            const query = req.query['query'] as string | undefined;
+            const limit = parseInt((req.query['limit'] as string) ?? '10');
```

Comment:
@sudhanshug16 forgot to ask a question on this limit parameter? What was your use case here? It doesn’t seem very useful without a pagination interface

Reply:
I wanted to limit the size of the output when paired with `query`. I agree it doesn't make sense without a pagination interface.
Removed it.

Reply:
@khaliqgant, gentle bump.

Would really appreciate it if this was merged asap, my launch depends on it :)

Reply:
No problem. Merging now and will deploy shortly!

Reply:
This has been deployed @sudhanshug16